packages/main/src/TimePickerClockTemplate.tsx

```diff
@@ -9,74 +9,73 @@ export default function TimePickerClockTemplate(this: TimePickerClock) {
 			aria-hidden="true"
 			class={{
 				"ui5-tp-clock": true,
-				"ui5-tp-clock-inner": this.showInnerCircle,
 				"ui5-tp-clock-active": this.active,
 			}}
 		>
 
 			<div data-label={this.label} class="ui5-tp-clock-dial"></div>
 
-			<div>
+			<div class="ui5-tp-clock-items">
```

Comment:
maybe not part of the scope of the task and this change, but I think it would be better to create separate templates for the smaller components in the future (for all of our components), for better readability and modularity.. 

It would look something like:

```tsx
export default function TimePickerClockTemplate(this: TimePickerClock) {
	return (
		<div
			id={`${this._id}`}
			onDragStart={this.noop}
			onDrop={this.noop}
			aria-hidden="true"
			class={{
				"ui5-tp-clock": true,
				"ui5-tp-clock-active": this.active,
			}}
		>
			{ ClockTemplate.call(this) }
			{ ClockItemsTemplate.call(this) }
			{ ClockMarkersTemplate.call(this) }
			{ ClockCoverTemplate.call(this) }
		</div>
	);
}
```

We can discuss it offline